.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
+enableGlobalCache: true
```

Comment:
Another useful setting is `nmMode: hardlinks-global`, which will make Yarn disk-space efficient by placing `hardlinks` from `node_modules` to the central store. It will also make recurrent installs even faster than with `compressionLevel: 0`.